api/src/main/java/org/apache/gravitino/credential/S3SecretKeyCredential.java

```diff
@@ -44,13 +45,17 @@ public class S3SecretKeyCredential implements Credential {
    * @param secretAccessKey The S3 static secret access key.
    */
   public S3SecretKeyCredential(String accessKeyId, String secretAccessKey) {
-    Preconditions.checkNotNull(accessKeyId, "S3 access key Id should not null");
-    Preconditions.checkNotNull(secretAccessKey, "S3 secret access key should not null");
-
+    validate(accessKeyId, secretAccessKey, 0);
```

Comment:
do you have a strong option to change this?  it's part of the credential API
```

  /**
   * Returns the expiration time of the credential in milliseconds since the epoch, 0 means not
   * expire.
   *
   * @return the expiration time as a long.
   */
  long expireTimeInMs();
```

Reply:
> do you have a strong option to change this? it's part of the credential API

Because different people have differing viewpoints about which numbers represent permanence, it's just a suggestion.

Reply:
I prefer to leave it as zero to keep compatibility. WDYT?

Reply:
Ok, maintaining the status quo is also acceptable to me.